manifests/init.pp

```diff
@@ -8,6 +8,8 @@
 #  [*version*]              - Version of StackStorm package to install (default = 'present')
 #                             See the package 'ensure' property:
 #                             https://puppet.com/docs/puppet/5.5/types/package.html#package-attribute-ensure
+#  [*release*]              - Release repository to enable. 'stable', 'unstable'
```

Comment:
I think `repository` or similar would be more more accurate for naming comparing to `release` and also consistent across other repos.

Release could be confused with package version revision/release number.

Reply:
i got `release` from the st2-packages "one-liner" scripts: https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-el7.sh#L14

absolutely can change to `repository` though